.pending/improvements/store/_4751-update-genutil

```diff
@@ -0,0 +1 @@
+#4751 update `x/genutils` to match module spec
```

Comment:
@alexanderbez I didn't know under which section should this go

Reply:
Mhhhh, either:

1. No tag at all
2. We create a `modules` tag
3. We create a tag for each module (e.g. `genutil`)

Reply:
I'd go for `2`